src/index.js

```diff
@@ -9,11 +9,13 @@ let todosArray = localStorage.getItem('todos') ? JSON.parse(localStorage.getItem
 const getValueFromInput = () => {
   const todoValue = inputTodo.value;
   inputTodo.value = '';
+  alert(todoValue);
   return todoValue;
 };
```

Comment:
- [ ] kindly remove the alert function which you have used in your application because it is not professional, Alert dialogs can interrupt the user's flow and may be considered intrusive. you can alternatively create a modal to handle this effect.